x-pack/plugins/security_solution/public/overview/components/events_by_dataset/index.tsx

```diff
@@ -138,16 +138,16 @@ const EventsByDatasetComponent: React.FC<Props> = ({
   );
 
   const [filterQuery, kqlError] = useMemo(() => {
```

Comment:
`nit` : Here `kqlError` will always be undefined. Do you think it should be removed?

Reply:
2 things:

1. I didn't touch this code, and I thought I already touched too much out of scope changes in other places so cleaning up even more was probably unnecessary to not further delay this already super delayed ticket :)

2. Why do you think this will return `undefined`, can you elaborate?

Reply:
Sorry.. This is my mistake. Please ignore this comment.